core/broadcast.go

```diff
@@ -241,7 +241,8 @@ func senderQueueSize(nodes int) int {
 	if nodes > maxQueueSize {
 		return maxQueueSize
 	}
-	return nodes
+	// we have 3 steps
+	return nodes * 3 //nolint:gomnd
```

Comment:
The 3 here seems a big magical. Can we maybe make it a parameter? Will it always be 3?

Reply:
It's related to the number of steps in the DKG. It will go away in the planned refactoring of the DKG and broadcast anyway.